app/models/repository.rb

```diff
@@ -20,6 +20,10 @@ def self.license_options
      "Attribution - Non-Commercial - No Derivatives"]
   end
 
+  def user_username
```

Comment:
ah, so that's a valid-ish failure. The test was assuming (incorrectly) that it could just set the title of `fixture(:one)` to the title of `fixture(:two)` and cause a failure. That _was_ the case because the uniqueness check is for uniquness per user_username. After you add user_username, the uniqnuess validation passes. The correct way to fix this isn't to rewrite this method, although I agree it's weird that this exists at all and should be removed later. I've made an issue to track it https://github.com/uOttawa-Makerspace/MakerSpaceRepo/issues/58